client/my-sites/domains/domain-management/edit/domain-types/registered-domain-type.jsx

```diff
@@ -0,0 +1,149 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { Card } from '@automattic/components';
+import VerticalNav from 'components/vertical-nav';
+import { domainAddNew } from 'my-sites/domains/paths';
+import { recordTracksEvent, recordGoogleEvent } from 'state/analytics/actions';
+import MaterialIcon from 'components/material-icon';
+import { withLocalizedMoment } from 'components/localized-moment';
+import VerticalNavItemMulti from '../vertical-nav/item-multi';
+
+class RegisteredDomainType extends React.Component {
+	getPickCustomDomainNavItem() {
+		return (
+			<VerticalNavItemMulti
+				path={ domainAddNew( this.props.selectedSite.slug ) }
+				onClick={ this.handlePickCustomDomainClick }
+				materialIcon="search"
+				text={ this.props.translate( 'Pick a custom domain' ) }
+				description={ this.props.translate( 'Register or transfer custom domain name' ) }
+			/>
+		);
+	}
+	getChangeDnsNavItem() {
+		return (
+			<VerticalNavItemMulti
+				path={ domainAddNew( this.props.selectedSite.slug ) }
+				onClick={ this.handlePickCustomDomainClick }
+				materialIcon="language"
+				text={ this.props.translate( 'Change your nameservers & DNS records' ) }
+				description={ this.props.translate( 'Destination: somewhere' ) }
+			/>
+		);
+	}
+
+	getEmailNavItem() {
+		return (
+			<VerticalNavItemMulti
+				path={ domainAddNew( this.props.selectedSite.slug ) }
+				onClick={ this.handlePickCustomDomainClick }
+				materialIcon="email"
+				text={ this.props.translate( 'Manage your email accounts' ) }
+				description={ this.props.translate( '3 G Suite accounts' ) }
+			/>
+		);
+	}
+
+	getContactInformationNavItem() {
+		return (
+			<VerticalNavItemMulti
+				path={ domainAddNew( this.props.selectedSite.slug ) }
+				onClick={ this.handlePickCustomDomainClick }
+				materialIcon="chrome_reader_mode"
+				text={ this.props.translate( 'Update your contact information' ) }
+				description={ this.props.translate( 'Privacy protection: on' ) }
+			/>
+		);
+	}
+
+	getTransferYourDomainNavItem() {
+		return (
+			<VerticalNavItemMulti
+				path={ domainAddNew( this.props.selectedSite.slug ) }
+				onClick={ this.handlePickCustomDomainClick }
+				materialIcon="cached"
+				text={ this.props.translate( 'Transfer your domain' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **21** times:
`translate( 'Adding your domain transfer' )` *ES Score: 7*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/new-domain-management-status-screen)